src/main/java/duke/Task.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.TextStyle;
+import java.util.Locale;
+/**
+ * Represents a Task item with a description.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public Task(String description) {
```

Comment:
Perhaps a new line between field and method declaration would look be more readable?